src/service/vaults.ts

```diff
@@ -261,6 +261,7 @@ export const watchVaultFactory = (netconfigUrl: string) => {
       );
       assert(instanceEntry, 'Missing VaultFactory from agoricNames.instances');
       useVaultStore.setState({ vaultFactoryInstanceHandle: instanceEntry[1] });
+      return;
```

Comment:
is this necessary?

Reply:
I figured it won't change, so better to not leave the follower open.

Reply:
how does it change anything about the follower? `return` at the end of a function is a noop

Reply:
It's inside the for-loop, I think that should work? We'll revisit once we get the new casting improvements anyway though

Reply:
shoot, reading from limited context in diff.